source/plugins/go/src/telemetry.go

```diff
@@ -194,6 +194,10 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 				if fbitTailMemBufLimitMBs != "" {
 					telemetryDimensions["FbitMemBufLimitSizeMBs"] = fbitTailMemBufLimitMBs
 				}
+				telemetryDimensions["PromFbitChunkSizeKB"] = os.Getenv("AZMON_FBIT_CHUNK_SIZE")
+				telemetryDimensions["PromFbitBufferSizeKB"] = os.Getenv("AZMON_FBIT_BUFFER_SIZE")
```

Comment:
same case here . This can be KB or MB. In case of the default, it will be KB else its MB.

Reply:
since the values now have the units we can remove the mention of KB/MB in key names